snippets/ztp_example-sno.yaml

```diff
@@ -12,95 +12,140 @@ spec:
-        # ../policygentemplates/example-sno-site.yaml will apply to all clusters with 'sites: "example-sno"'
-        # Normally this should match or contain the cluster name so it only applies to a single cluster
-        sites: "example-sno"
-      clusterNetwork:
-        - cidr: 1001:1::/48
-          hostPrefix: 64
-      machineNetwork:
-        - cidr: 1111:2222:3333:4444::/64
-      serviceNetwork:
-        - 1001:2::/112
-      additionalNTPSources:
-        - 1111:2222:3333:4444::2
-      # Initiates the cluster for workload partitioning. Setting specific reserved/isolated CPUSets is done via PolicyTemplate
-      # please see Workload Partitioning Feature for a complete guide.
-      cpuPartitioningMode: AllNodes
-      # Optionally; This can be used to override the KlusterletAddonConfig that is created for this cluster:
-      #crTemplates:
-      #  KlusterletAddonConfig: "KlusterletAddonConfigOverride.yaml"
-      nodes:
-        - hostName: "example-node1.example.com"
-          role: "master"
-          # Optionally; This can be used to configure desired BIOS setting on a host:
-          #biosConfigRef:
-          #  filePath: "example-hw.profile"
-          bmcAddress: "idrac-virtualmedia+https://[1111:2222:3333:4444::bbbb:1]/redfish/v1/Systems/System.Embedded.1"
-          bmcCredentialsName:
-            name: "example-node1-bmh-secret"
-          bootMACAddress: "AA:BB:CC:DD:EE:11"
-          # Use UEFISecureBoot to enable secure boot
-          bootMode: "UEFI"
-          rootDeviceHints:
-            wwn: "0x11111000000asd123"
-            # example of diskPartition below is used for image registry (check ImageRegistry.md for more details), but it's not limited to this use case
-          #        diskPartition:
-          #          - device: /dev/disk/by-id/wwn-0x11111000000asd123 # match rootDeviceHints
-          #            partitions:
-          #              - mount_point: /var/imageregistry
-          #                size: 102500
-          #                start: 344844
-
-          nodeNetwork:
+  - clusterName: "example-sno"
+    networkType: "OVNKubernetes"
+    # installConfigOverrides is a generic way of passing install-config
+    # parameters through the siteConfig.  The 'capabilities' field configures
+    # the composable openshift feature.  In this 'capabilities' setting, we
+    # remove all but the marketplace component from the optional set of
+    # components.
+    # Notes:
+    # - OperatorLifecycleManager is needed for 4.15 and later
+    # - NodeTuning is needed for 4.13 and later, not for 4.12 and earlier
+    installConfigOverrides: | 
+      {
+        "capabilities": {
+          "baselineCapabilitySet": "None",
+          "additionalEnabledCapabilities": [
+            "NodeTuning",
+            "OperatorLifecycleManager"
+          ]
+        }
+      }
+    # It is strongly recommended to include crun manifests as part of the additional install-time manifests for 4.13+.
```

Comment:
What does 4.13+ refer to? I assume you mean OpenShift. If so, state explicitly and also use the convention of "{product-title} 4.13 or later."

Reply:
This reflects a YAML file created by dev so I cannot change this as it needs to be the same as one downloaded as part of zip file.